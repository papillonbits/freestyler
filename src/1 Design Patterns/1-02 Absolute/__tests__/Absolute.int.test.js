import { regular } from './Absolute.int.story'

describe('<Absolute />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
