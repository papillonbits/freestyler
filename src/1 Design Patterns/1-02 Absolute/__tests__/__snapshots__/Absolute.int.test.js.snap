// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Absolute /> Render must match regular() 1`] = `
<div>
  <div
    class="absolute"
  >
    <h1>
      Absolute
    </h1>
    <div
      class="absolute__positioned"
    >
      <span
        class="absolute__absolute"
      >
        Sized Absolute
      </span>
    </div>
  </div>
</div>
`;
